sdk/platform/http_client/curl/src/curl_client.cpp

```diff
@@ -99,6 +99,10 @@ static void ParseHeader(std::string const& header, std::unique_ptr<Response>& re
 
   auto headerName = std::string(start, end);
   start = end + 1; // start value
+  if (start < header.end() && *start == ' ')
```

Comment:
What about multiple whitespace? An what about other whitespace characters like horizontal tab? We should see what the HTTP RFC states about this:
https://tools.ietf.org/html/rfc7230#section-3.2

> header-field   = field-name ":" OWS field-value OWS

> OWS            = *( SP / HTAB )
                    ; optional whitespace

Reply:
@ahsonkhan updated.